src/ngraph/runtime/cpu/pass/cpu_rnn_fusion.cpp

```diff
@@ -219,12 +219,12 @@ void ngraph::runtime::cpu::pass::LSTMFusion::construct_onnx_lstmcell_fprop()
             lstm_ht_ct_output, Coordinate{batch_size, 0}, Coordinate{(2 * batch_size), dic});
 #endif
         // find the user's for {ht} and replace them with lstm_goe_0
```

Comment:
Find the user's what? Find the users for `{ht}`? Users might be the wrong word here if users means "framework node" (nodes?) or something like that.